.github/workflows/ci.yml

```diff
@@ -93,10 +100,11 @@ jobs:
         with:
           python-version-file: .tool-versions
       - run: pip install poetry==$(cat .tool-versions | grep poetry | cut -d" " -f2)
+        working-directory: .
```

Comment:
I think CI is failing due to it. what's the reason you need to override `working-directory` here? I think all the steps should be in `packages/python`, or I'm missing something